### PR TITLE
Added italian translation

### PR DESCRIPTION
**Pull request by Raffox97**
Don't know if some parenthesis are right.

### CODE REVIEW

File: web/app/configs/lang.json [hunk 1]
```diff
@@ -125,4 +125,67 @@
         "UA": "Ukraine"
     }
   }
```
Comment (corradio) [inline web/app/configs/lang.json]: Needs a comma

**Comment by corradio** [comment]
Previews:

image
image
image
image
image
image

**Comment by corradio** [comment]
The text for hydro storage is quite long, can you make it shorter?

**Comment by Raffox97** [comment]
Done. Let me know if there're any other errors

**Comment by corradio** [comment]
do all the screenshots make sense?

**Comment by Raffox97** [comment]
Yes

**Comment by corradio** [comment]
Then I'm merging!

**Comment by Raffox97** [comment]
Perfect

**Pull request merged**